cmd/ecrecover.go

```diff
@@ -44,15 +44,15 @@ var signCmd = &cobra.Command{
 		publicKey := privateKey.Public()
 		publicKeyECDSA := publicKey.(*ecdsa.PublicKey)
 		addressSig := crypto.PubkeyToAddress(*publicKeyECDSA).Hex()
-		logrus.Info("sign with address %s\n", addressSig)
+		logrus.Info("sign with address ", addressSig)
 
 		input := message
 		signature, _, err := util.PersonalSign(input, privateKey)
 		if err != nil {
 			log.Fatal(err)
 		}
 
-		logrus.Info("signature", signature, "length", len(signature))
+		logrus.Info("signature ", signature, "length ", len(signature))
```

Comment:
The  format might be ugly, the recommended way to use is:
`logrus.WithFields(...).Info(...)`